week-4/D-methods/exercise-3.js

```diff
@@ -8,11 +8,11 @@ var person = {
     name: "Alice",
     age: 25,
     currentAddress: "Glasgow",
-    changeAddress: (newAddress) {
+    changeAddress: function (newAddress) {
         currentAddress = newAddress;
```

Comment:
There's a bug here. Try to debug it by actually calling your method `changeAddress`.

Reply:
@foaly-nr1, thanks for point that out, I spotted the bug and I think I have fixed it, will be pushing the correction shortly